kano_world/functions.py

```diff
@@ -16,24 +16,23 @@
 from .connection import request_wrapper, content_type_json
 from .session import KanoWorldSession
 
+import mercury
 
 glob_session = None
+http_verbose = False
+kw_url = 'https://worldapi-dev.kano.me'
```

Comment:
Should we not use `API_URL` from `kano_world.config` instead? We can update the URL there.

Reply:
👍

Reply:
Fixed